example/javalib/linting/3-palantirformat/build.mill

```diff
@@ -0,0 +1,18 @@
+package build
+
+import mill._
+import mill.javalib.palantirformat._
+
+object `package` extends RootModule with PalantirFormatModule {
+}
+
+/** Usage
+
+> ./mill palantirformat                                                 # format all Java source files
+...formatting java sources ...
+
+> ./mill mill.javalib.palantirformat.PalantirFormatModule/ __.sources   # format all Java source files in existing modules
+...formatting java sources ...
+*/
```

Comment:
We should show `--check` failing before calling `palantirformat`, and then passing after

Reply:
Using `--check` results in a failure.
```sh
./mill example.javalib.linting[3-palantirformat].local
```
```
 Incomplete argument --check <bool> is missing a corresponding value
  Expected Signature: palantirformat
    --check <bool>
    sources <str>...
  

  ,)
```

Reply:
You need to make it `check: mainargs.Flag` for it to not require a boolean after